### DISCUSSION TITLE
推荐？

### DISCUSSION THREAD

**Discussion by ohay5128** [xiaocao666tzh/Airport-Checkin#4] [Q&A] [answered]
有哪些鸡场可以白嫖呢，大佬推荐几个嘛

**Comment by xiaocao666tzh** [comment, marked as answer]
这种东西，不做推荐。本仓库仅做技术交流，不提供任何翻墙方面的讨论!

**Comment by ohay5128** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by QxaSJTU** [comment]
你好，action的yml文件的仓库被删了，请问能提供一下yml文件吗？谢谢

**Comment by ohay5128** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
